Tests/PSGetFindModule.Tests.ps1

```diff
@@ -24,12 +24,13 @@ function SuiteSetup {
     $script:MyDocumentsModulesPath = Get-CurrentUserModulesPath
     $script:PSGetLocalAppDataPath = Get-PSGetLocalAppDataPath
     $script:DscTestModule = "DscTestModule"
+    $script:PrereleaseTestModule = "TestPackage"
```

Comment:
Since this is not resolved yet, should we keep it in active state?

---
In reply to: [126798793](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r126798793) [](ancestors = 126798793)